apps/synapse-portal-framework/src/components/standards/StandardsContributeToTheRegistry.tsx

```diff
@@ -1,31 +1,18 @@
 import { Button, Stack, Typography } from '@mui/material'
-import { CTASectionWrapper } from 'synapse-react-client'
 
 const StandardsContributeToTheRegistry = () => {
-  const title = 'Contribute to the registry'
   const subtitle =
     'Help expand our catalog of biomedical research data standards by contributing your own. Submit your standards through our easy submission process to share them with the community and advance data harmonization. Join us in shaping the future of biomedical research!'
   const buttonText = 'Add a STANDARD to the registry'
   const buttonLink =
     'https://github.com/bridge2ai/b2ai-standards-registry/tree/main?tab=readme-ov-file#submitting-a-new-standard'
 
-  const content = (
-    <Stack sx={{ gap: '20px' }}>
-      <Typography
-        variant="headline2"
-        sx={{
-          fontWeight: 700,
-          fontSize: '24px',
-          lineHeight: 'normal',
-          textTransform: 'uppercase',
-        }}
-      >
-        {title}
-      </Typography>
```

Comment:
Let the `SectionLayout` on the portal homepage handle rendering the title (with link support)